lxd/db/openfga/openfga.go

```diff
@@ -149,6 +150,40 @@ func (o *openfgaStore) Read(ctx context.Context, s string, key *openfgav1.TupleK
 			},
 		}
 
+	case "instance":
```

Comment:
We can do yes. The reason I didn't do this is because these are relations in the model and not necessarily entity types. We have used the same name in our model though, and I think it makes sense to always do so, so I'll change it.

Reply:
its useful to see where they are used.